README.md

```diff
@@ -7,6 +7,16 @@ Talk to ChatGPT via your favourite Matrix client!
 
 This is an unofficial Matrix bot that uses https://github.com/transitive-bullshit/chatgpt-api to access the unofficial ChatGPT API.
 
+It is worth reading the [authentication instructions](https://www.npmjs.com/package/chatgpt#usage) for the unofficial ChatGPT API.
+
+Your user-agent and IP address must match from the real browser window you're logged in with to the one you're using for ChatGPTAPI. This means that you currently can't log in with your laptop and then run the bot on a server or proxy somewhere.
```

Comment:
Really appreciate the detailed instructions here!

Is there any way of running the bot on a headless ubuntu server and running the CAPTCHA? I assume forwarding X over ssh would be required?